lib/ae_mdw_web/views/aexn_view.ex

```diff
@@ -43,7 +43,7 @@ defmodule AeMdwWeb.AexnView do
            height: AeMdw.Blocks.height(),
            last_tx_hash: Txs.tx_hash(),
            last_log_idx: AeMdw.Contracts.log_idx(),
-           amount: integer()
+           amount: float()
```

Comment:
It is an integer(), in ættos

Reply:
Type checking is complaining because it's being specified as a `float()` on the `aex9_event_balance` record:
https://github.com/aeternity/ae_mdw/blob/1bedf79e4f95b0aeb2043a9614e128138c72fc29/lib/ae_mdw_web/views/aexn_view.ex#L100
https://github.com/aeternity/ae_mdw/blob/1bedf79e4f95b0aeb2043a9614e128138c72fc29/lib/ae_mdw/db/model.ex#L384